### DISCUSSION TITLE
doubt in faster rcnn config

### DISCUSSION THREAD

**Discussion by zhaoxin111** [open-mmlab/mmdetection#5470] [Q&A]
![image](https://user-images.githubusercontent.com/9295202/123618870-2e83ed00-d83b-11eb-8d42-c4b2f1f8c458.png)
![image](https://user-images.githubusercontent.com/9295202/123618986-4fe4d900-d83b-11eb-8ea1-cd31908fdcb5.png)

According to the logic of the _bbox_forward method, only the first four feature layers are responsible for bbox feature representation. I check the config of faster rcnn in detectron2, all 5 feature maps are responsible for the proposal feature representation. Why mmdet only use 4 feature maps in  bbox_roi_extractor?

**Comment by ZwwWayne** [comment]
The five feature maps are used for proposal generation. But for RoI feature extraction we only use the 4 feature maps.

**Reply by zhaoxin111** [reply]
Why？
